projects/packages/my-jetpack/global.d.ts

```diff
@@ -31,7 +31,7 @@ type ScanItem = {
 	checked: boolean;
 	name: string;
 	slug: string;
-	threats: string[];
+	threats: object[];
```

Comment:
Can we define the object here, or the structure is unknown?

Reply:
Yes, will do. 👍